website/source/docs/builders/alicloud-ecs.html.md

```diff
@@ -18,7 +18,7 @@ customized images based on an existing base images.
 
 The following configuration options are available for building Alicloud images.
 In addition to the options listed here,
-a [communicator](/docs/templates/communicator.html) can be configured for this
+a [communicator](../templates/communicator.html) can be configured for this
```

Comment:
For reference, this is the result of https://github.com/jsoref/hashicorp-packer/commit/4f8abb816467c9ca0908df2d9f05a8e072800027